modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -36,16 +36,18 @@ PERF_TEST_P_(AddPerfTest, TestPerformance)
     cv::GComputation c(GIn(in1, in2), GOut(out));
 
     // Warm-up graph engine:
-    c.apply(gin(in_mat1, in_mat2), gout(out_mat_gapi), std::move(compile_args));
+    auto cc = c.compile(cv::descr_of(in_mat1), cv::descr_of(in_mat2), std::move(compile_args));
+    cc(in_mat1, in_mat2, out_mat_gapi);
```

Comment:
let's stick to common pattern : i.e. use `gin()` and `gout()` explicitly. 
i.e. 
```cpp
auto cc = c.compile(descr_of(gin(ginin_mat1, in_mat2)), std::move(compile_args));
cc(gin(in_mat1, in_mat2), gout(out_mat_gapi));
```

please note that no need to qualify `gin`, `gout` , `descr_of` with namespace (`cv::`) as they are in the same namespace as their actual argument (i.e. `cv::Mat`) and thus are found via [ADL](https://en.cppreference.com/w/cpp/language/adl)

Reply:
please apply this to all the changed calls

Reply:
Fixed